runtime/lua/vim/keymap.lua

```diff
@@ -0,0 +1,127 @@
+local keymap = {}
+
+--- Add a new keymap.
+--- Examples:
+--- <pre>
+---   -- You can make keymaps worth lua functions
```

Comment:
"Lua" is a proper noun, so in documentation it should always be capitalized.

```suggestion
---   -- You can make keymaps worth Lua functions
```